capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -37,6 +30,12 @@
 
 public class Utility {
   public static Firestore getFirestoreDb() throws IOException {
+
+    // Firebase already initialize
+    if (!FirebaseApp.getApps().isEmpty()) {
```

Comment:
What does this do?

Reply:
Prevent a duplicate error. If Firebase its already initialize then it will throw an error of already being instantiated on the Backend Logs. This just checks if Firebase is already initialize.